Sources/SQLKit/Builders/SQLUnionBuilder.swift

```diff
@@ -0,0 +1,25 @@
+public final class SQLUnionBuilder: SQLQueryBuilder {
+    public var query: SQLExpression {
+        return self.union
+    }
+
+    public var union: SQLUnion
+    public var database: SQLDatabase
+
+    public init(on database: SQLDatabase,
+                _ args: [SQLSelectBuilder],
+                all: Bool = false) {
+        self.union = .init(args.map { $0.select }, all: all)
+        self.database = database
+    }
+}
+
+extension SQLDatabase {
+    public func union(_ args: SQLSelectBuilder...) -> SQLUnionBuilder {
+        SQLUnionBuilder(on: self, args)
+    }
+
+    public func unionAll(_ args: SQLSelectBuilder...) -> SQLUnionBuilder {
+        SQLUnionBuilder(on: self, args, all: true)
+    }
```

Comment:
I'm not sure if `unionAll` is great but my thinking was that for the main API it'd be better if the `all` was more obvious than a trailing parameter.

Of course, it could instead be `union(all: true, _ args: ...)` but it feels like a bit of a weird interface in general with a boolean flag that isn't very descriptive.

Maybe it should be `union(allowDuplicates: true, _ args ...)`? Although I don't think there's API precedent to interpret SQL semantics rather than just pass through the raw syntax in a swifty way.

Reply:
I think I've convinced myself to change this to `union(all: true, _ args: ...)` (and move `all:` to the front in `SQLUnion` accordingly).

What do you think?

Reply:
How about `union(all: ...)`?